dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -1320,7 +1330,159 @@ brokerLocale=en_US
 
 				Restart-Service $using:brokerServiceName
             }
-        }
+		}
+		
+		Script RegisterCam
+		{
+			DependsOn  = @("[Script]Install_Auth_file")
+			GetScript  = { @{ Result = "RegisterCam" } }
+
+            TestScript = { 
+				if ( $env:CAM_USERNAME -and $env:CAM_PASSWORD -and $env:CAM_TENANTID -and $env:CAM_URI -and $env:CAM_DEPLOYMENTID)
+				{
+					return $true
+				} else {
+					return $false
+				}
+			}
+
+            SetScript  = {
+				##
+				# Do this so SSL Errors are ignored
+				add-type @"
+	using System.Net;
+	using System.Security.Cryptography.X509Certificates;
+	public class TrustAllCertsPolicy : ICertificatePolicy {
+		public bool CheckValidationResult(
+			ServicePoint srvPoint, X509Certificate certificate,
+			WebRequest request, int certificateProblem) {
+			return true;
+		}
+	}
+"@
+				[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
```

Comment:
Which context does this get changed for? Just the current executing environment? PS Session?